### PR TITLE
fix: hidden vs disabled

### PR DESCRIPTION
**Pull request by drdelambre**
added a hidden state so that the user knows an option is available eventually, just not yet because some status hasn't been unlocked. also adds a patch to the schedule dropdown for when users wanna overwrite an existing task

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 51 | :red_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=3914)

File: src/flows/pipes/Schedule/view.tsx [hunk 1]
```diff
@@ -300,12 +301,42 @@ const Schedule: FC<PipeProp> = ({Context}) => {
         actions: [
           {
             title: 'View Run History',
-            menu: <History tasks={data.task.id ? data.task : data.task} />,
+            menu: <History tasks={data.task.id ? [data.task] : data.task} />,
+          },
+          {
+            title: 'Overwrite Existing Task',
+            disable: () => !latestTask || !hasChanges,
```
Comment (asalem1) [inline src/flows/pipes/Schedule/view.tsx:308]: should this be hidden? I don't get the distinction

File: src/flows/pipes/Visualization/view.tsx [hunk 2]
```diff
@@ -120,12 +120,14 @@ const Visualization: FC<PipeProp> = ({Context}) => {
           {
             title: 'Download as Image',
             action: () => downloadAsImage(id),
-            disable: !isFlagEnabled('pdfImageDownload'),
+            hidden: !isFlagEnabled('pdfImageDownload'),
+            disable: !dataExists,
```
Comment (asalem1) [inline src/flows/pipes/Visualization/view.tsx:124]: why?

**Review by asalem1 — COMMENTED** [review]
I've got some general questions about existing disabled pipes, how those will be impacted by the transition from disabled -> hidden, and how we make the distinction of when to disable vs hide (plus in general, the disabling of the pdf / img output for users that want to save a PDF of an unrun notebook)

**Pull request merged**